virttest/guest_agent.py

```diff
@@ -725,3 +725,35 @@ def fsthaw(self, check_status=True):
                     raise
             return ret
         return -1
+
+    def openfile(self, args):
+        """
+        args:specify file's path and mode option when open file
+        """
+        cmd = "guest-file-open"
+        if self.check_has_command(cmd):
+            return self.cmd(cmd, args, debug="Ture", success_resp="False")
```

Comment:
@balamuruhans Thanks for your view.  I used success_resp="True" to replace success_resp="False" and debug = "True" replace debug = "Ture".  I am trying to integrate 4 functions into a function.   I am debugging it.